src/Tools/Test Framework/Test Runner/src/TestMethodLine.Table.al

```diff
@@ -106,6 +106,18 @@ table 130450 "Test Method Line"
         {
             Editable = false;
         }
+        field(13; "Test Type"; Option)
```

Comment:
Should we introduce an enum to the platform? Then we can reuse it.

Reply:
Good question. Decided to go with `Option` because it will not be used anywhere else, and we will have the flexibility to easily change it.

Especially with the latest commit, this is only converted inside one procedure, it is probably ok.

What do you think?